tools/src/rpc-schema/schema.json

```diff
@@ -0,0 +1,4420 @@
+{
```

Comment:
Why to have this file versioned? To me looks like the product of this tool and it might change. Sounds like we need to have a CI that on release publish this file somewhere

Reply:
I will rework this and it will just write it to stdout. Then let the CI call this tool and add it as a release attachment to the latest release when one is created.